src/solver/conv/conv_asm_1x1u_stride2.cpp

```diff
@@ -511,7 +511,7 @@ bool ConvAsm1x1UV2::IsApplicable(const ExecutionContext& ctx,
         return false;
 
     const std::string name = ctx.GetStream().GetDeviceName();
-    if(name.find("gfx8") == std::string::npos && name.find("gfx9") == std::string::npos)
+    if(!(StartsWith(name, "gfx8") || StartsWith(name, "gfx90")))
```

Comment:
It might be a good idea to clean up and remove all the code for `gfx8`. Not in this PR of course.